responsibleai/responsibleai/_interfaces.py

```diff
@@ -91,10 +91,22 @@ class CausalPolicyTreeLeaf:
     treatment: str
 
 
+class ComparisonTypes:
```

Comment:
Let's check with @imatiach-msft , current ui and error analysis line up with different set of enum.
We need to standardize to one set, I am not 100% sure about impact.

Reply:
Discussed offline, agreed that the causal changes can go in first with changes to error analysis to standardize later if necessary